R/reporter-progress.R

```diff
@@ -18,6 +18,8 @@ ProgressReporter <- R6::R6Class("ProgressReporter",
     show_praise = TRUE,
     min_time = 0.1,
     start_time = NULL,
+    last_update = NULL,
+    update_interval = 0.1,
```

Comment:
It is a small thing, but I would set this to `NULL` here. It is set by `initialize` anyway, and then this magic number is misleading.